querydsl-sql/src/test/java/com/querydsl/sql/types/TypeTest.java

```diff
@@ -45,7 +45,29 @@
 
     @Override
     public Object invoke(Object proxy, Method method, Object[] args) throws Throwable {
-        if (method.getName().startsWith("get")) {
+        if (method.getName().equals("wasNull")) {
```

Comment:
I'd consider the inverted order only for known nullable types and not as a general practice, since having the non-null visible in the access is also valuable and at least for me this order is more readable.